plugins/cost-insights/src/index.ts

```diff
@@ -18,7 +18,8 @@ export { plugin } from './plugin';
 export * from './client';
 export * from './api';
 export * from './components';
-export { useCurrency } from './hooks';
+export { useCurrency, CurrencyContext } from './hooks';
 export * from './types';
 export * from './utils/tests';
 export * from './utils/duration';
+export * from './utils/alerts';
```

Comment:
😞

Reply:
Also, I think we need to figure out a more consistent approach for handling public exports at some point in the near future. At the moment, we have about three different approaches - wholesale export from index (types), partial export from index (components), and export from file (utils).

Reply:
Use this: https://backstage.io/docs/architecture-decisions/adrs-adr004 ! 😁